packages/pages/src/upgrade/upgrade.ts

```diff
@@ -12,8 +25,20 @@ interface UpgradeArgs {
 
 const handler = async (args: UpgradeArgs) => {
   const scoped = { scope: args.scope || "" };
+  const source = path.resolve(scoped.scope);
   const projectStructure = await ProjectStructure.init(scoped);
-  await updatePages(projectStructure);
+  await updateSitesComponents(source);
+  await updateDevDependencies(source);
+  checkLegacyMarkdown(source);
+  replacePagesSlashComponentsImport(source);
+  replaceSitesComponentsImports(source);
+  removeFetchImport(source);
+  updatePackageScripts(source);
+  updatePackageEngines(source);
+  checkNodeVersion();
+  await updatePages(source);
+  // await updatePagesComponents(source);
```

Comment:
This will be enabled and the updateSitesComponent call removed once pages-components is released